Makefile

```diff
@@ -1,15 +1,30 @@
 CHART_DIR	:= helm-charts
 GIT_VERSION	?= $(shell git describe --dirty=-unsupported --always --long --tags)
+DOCKER_NAME	?= cm
 HELM		?= docker run --rm -i \
 			--entrypoint="" \
 			--network host \
 			-e KUBECONFIG=/apps/.kube/$(notdir $(KUBECONFIG)) \
 			-v $(dir $(KUBECONFIG)):/apps/.kube/ \
 			-v $(PWD):/apps \
-			-v $(PWD)/repositories.yaml:/tmp/.config/helm/repositories.yaml \
-			-v $(PWD)/jetstack-index.yaml:/tmp/.cache/helm/repository/jetstack-index.yaml \
 			infoblox/helm:3.2.4-5b243a2 \
 			helm
+HELM_RA         ?= docker run -i \
+                        --entrypoint="" \
+                        --network host \
+			--name $(DOCKER_NAME) \
+                        infoblox/helm:3.2.4-5b243a2 \
+                        helm
```

Comment:
Since most of this is in common for all three commands, you could use a variable to define this part once and then construct each command by using the variable.

Reply:
Done